src/site/content/en/blog/coop-coep/index.md

```diff
@@ -29,6 +29,9 @@ feedback:
 
 **Updates**
```

Comment:
@kaycebasques I think a caution banner might be an overkill here, for example, in news reporting updates are usually just in italics. This might come up again so we should decide on a template for explicit update notes. Have we had something similar to this on web.dev before? How were updates handled on Web fundamentals?

Reply:
Bold SGTM. We haven't had a consistent policy in the past.

Reply:
Thanks for your input ✅